MyLibs/talk2screen.cpp

```diff
@@ -87,37 +87,10 @@ void PatchData::updatePattern()
 
 bool AreaData::initialize(vector<int> yDivideVec, string imgName)
 {
-	for (int i = 0; i < numPatches; i++)
-	{
-		vector<float> patchRect = rect;
-		switch (i) {
-		case 0:
-
-			break;// do nothing
-		case 1:
-		{
-			patchRect[0] -= patchRect[2] / 2; // minus half area width
-			break;
-		}
-		case 2:
-		{
-			patchRect[1] += patchRect[3] / 2;
-			break;
-		}
-		case 3:
-		{
-			patchRect[0] -= patchRect[2] / 2;
-			patchRect[1] += patchRect[3] / 2;
-			break;
-		}
-		default:;
-		}
-		patchRect[2] /= 2; // the width of patch is half of area width
-		patchRect[3] /= 2; // the height of patch is half of area width
-		PatchData patch(patchRect, yDivideVec[i]);
-		patch.initialize();
-		allPatches.push_back(patch);
-	}
+	vector<float> patchRect = rect;
```

Comment:
Actually, this file needs no edition since the numPatches has been set to 1.

Reply:
You are right. This function could be achieved by just doubling length and width.